libs/shared-atomic-design-components/src/lib/atoms/Chip.tsx

```diff
@@ -0,0 +1,14 @@
+import './Chip.scss'
+
+export interface ChipProps {
+  chip: string
+}
+
+const Chip = ({ chip }: ChipProps) => {
+  return (
+    <p key={chip} className="chip">
```

Comment:
I think this key should go where the "map" function is being called. I hope I didn't stay away from React enough to forget this simple rule 😓

Reply:
Sounds right. Don't worry, you're not losing your React chops to Angular already 🙌